app/views/posts/index.html.erb

```diff
@@ -1 +1,4 @@
-<h1> This is a list of posts</h1>
\ No newline at end of file
+<%= render partial: 'shared/user', locals: { user: @user } %>
+<%= render partial: 'shared/user_posts', locals: { posts: @user.posts, ren: 0 } %>
```

Comment:
- [ ] [REQUIRED I can see that you are rendering two partials here in the user's post index. However, we need to also adjust the design to match the wireframe provided. Kindly fix this as it is a project requirement.
### Your Design
<img width="1179" alt="Screenshot 2023-06-14 at 23 49 17" src="https://github.com/sanieni6/BlogApp/assets/58692590/8ab1e385-33f2-47a3-8136-328ee6451a24">

### Wireframe Provided
<img width="512" alt="Screenshot 2023-06-15 at 00 15 58" src="https://github.com/sanieni6/BlogApp/assets/58692590/444a69f4-7d01-4160-a5d1-1e913ff57f74">